tests/Resources/Page/ODH/ODHDashboard/ODHAdminUI.robot

```diff
@@ -0,0 +1,85 @@
+*** Settings ***
+Library         OpenShiftLibrary
+Resource        ../../LoginPage.robot
+
+*** Keywords ***
+Add OpenShift Groups To Data Science Administrators
+    [Documentation]  Add OpenShift groups to Data Science administrators
```

Comment:
Documentation is same as keyword name could you add `Add OpenShift groups to Data Science administrators From RHODS Dashboard` or something like this

Reply:
is it necessary? @bdattoma is it fine if I change it?